web/client/components/swipe/EffectSupport.jsx

```diff
@@ -0,0 +1,99 @@
+/*
+* Copyright 2020, GeoSolutions Sas.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
```

Comment:
This is the EffectSupport implementation for Openlayers ,it should go together with the other supports in `openlayers` directory.